src/main/java/seedu/allonus/contacts/ContactsManager.java

```diff
@@ -178,7 +209,7 @@ public static void contactsRunner(TextUi ui) {
                 editContact(userInput);
             } else {
                 printFormat(CONTACTS_INVALID_COMMAND_MESSAGE);
-                logger.log(Level.FINER, String.format("Invalid command to Contacts Manager: %s", userInput));
+                logger.log(Level.FINER, String.format(CONTACTS_INVALID_COMMAND_LOG_MESSAGE, userInput));
```

Comment:
I think for the final iteration, we should standardize how we log our messages.

Reply:
Yes agreed, thanks for pointing this out